compiler/rustc_error_codes/src/error_codes/E0492.md

```diff
@@ -3,10 +3,11 @@ A borrow of a constant containing interior mutability was attempted.
 Erroneous code example:
 
 ```compile_fail,E0492
+#![feature(const_refs_to_cell)]
```

Comment:
What is this about? Doesn't seem great to have a nightly-only example in the docs.

Reply:
It's the only way to trigger this error code now. Everything else causes a feature error

Reply:
So there's a feature error even for code that is still not accepted when the feature is enabled? Isn't that a bug?

Reply:
hmm... good point ^^ Well, tbh, we have this a lot of times. You get a feature error, but even with the feature the code won't compile. Just take any feature that is gated at the syntax level... you can still run into a typeck error later.

Reply:
I am surprised that this happens here though, given how fine-grained the const checking setup is.

Also, stable doesn't show the feature errors anyway, does it? So for a stable user this new example here is really rather confusing.

Reply:
yea... I looked into it, and it just works if I remove the feature gate, this must be a leftover from the original dataflow version of this PR